kibana/docker/Makefile

```diff
@@ -26,8 +26,9 @@ OPENDISTRO_REGISTRY ?= amazon
 
 ES_VERSION ?= $(shell ../bin/version-info --es)
 OPENDISTRO_VERSION ?= $(shell ../bin/version-info --od)
+OPENDISTRO_PLUGIN_VERSION ?= $(OPENDISTRO_VERSION).0
 IMAGE_TAG ?= $(OPENDISTRO_REGISTRY)/opendistro-for-elasticsearch-kibana
-OPENDISTRO_PLUGINS ?= opendistro-sql-kibana/sql-kibana-$(OPENDISTRO_VERSION).0.zip opendistro-anomaly-detection/opendistro-anomaly-detection-kibana-$(OPENDISTRO_VERSION).0.zip opendistro-security/opendistro_security_kibana_plugin-$(OPENDISTRO_VERSION).0.zip opendistro-alerting/opendistro-alerting-$(OPENDISTRO_VERSION).0.zip opendistro-index-management/opendistro_index_management_kibana-$(OPENDISTRO_VERSION).0.zip
```

Comment:
How did this work for 1.8.0 if is changed now?

Reply:
The Makefile string is not used by Dockerfile in Kibana.
We will make improvements on docker building process soon.